### PR TITLE
[Sprint: 44] XD-2712 Default LocalMessageBus for singlenode

### PR DESCRIPTION
**Pull request by dturanski**

### CODE REVIEW

File: spring-xd-messagebus-local/src/main/resources/META-INF/spring-xd/bus/local-bus.xml [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
 
-	<bean id="messageBus" class="org.springframework.xd.dirt.integration.bus.local.LocalMessageBus">
-		<property name="queueSize"
-			value="${xd.local.transport.named.queueSize: #{T(Integer).MAX_VALUE}}" />
-		<property name="poller">
-			<int:poller fixed-rate="${xd.local.transport.named.polling:1000}" />
-		</property>
-	</bean>
-
+	<beans profile="singlenode">
```
Comment (ericbottard) [inline spring-xd-messagebus-local/src/main/resources/META-INF/spring-xd/bus/local-bus.xml]: Why the need of a profile if it's only gonna be included when transport == local anyway?

Comment (dturanski) [inline spring-xd-messagebus-local/src/main/resources/META-INF/spring-xd/bus/local-bus.xml]: True, I don't think we need it.

Comment (ericbottard) [inline spring-xd-messagebus-local/src/main/resources/META-INF/spring-xd/bus/local-bus.xml]: And then, the next question is: why was the addition of that profile to JobPluginTests.java needed

Comment (dturanski) [inline spring-xd-messagebus-local/src/main/resources/META-INF/spring-xd/bus/local-bus.xml]: That's removed in the latest commit

**Comment by ericbottard** [comment]
Rebased and merged as 61060d157238329856f772730b2017cff93be009

**Pull request closed without merging**